### PR TITLE
activate the validation

### PR DESCRIPTION
**Pull request by Alzubair98**
in this pull request  i activate the validation in the contact form

### CODE REVIEW

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @Alzubair98  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [ ] When I submit the form it says "Form not found". Kindly fix this.
Screen Shot 2022-06-17 at 10 37 07 PM
_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alzubair98** [comment]
hello @zainsadaqat 
this project is just for Portfolio: validate contact form
not for the local storage the next project is for the local storage so i think its okay if you cant see the form yet

File: js/index.js [hunk 1]
```diff
@@ -323,3 +323,18 @@ window.addEventListener('load', () => {
     fullBody.classList.toggle('active');
   });
 });
+
+const form = document.getElementById('contact-information');
+
```
Comment (KDlamini) [inline js/index.js:328]: [OPTIONAL] When I submit the form, the error message shows even if the email is in lowercase. Kindly look into this and make sure that it only shows when the email contains uppercase letters.

image

**Review by KDlamini — APPROVED** [review]
### Hi @Alzubair98 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the project requirements. 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!🎊 🥳 

# Status: Approved ✔️ 

giphyapproved
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 1 above)
Comment (Alzubair98) [reply js/index.js:328]: hi 
thanks for the great feedback 
yeah i removed it 
i used (remove) instead of (toggle)

Comment (KDlamini) [reply js/index.js:328]: Fantastic job!!! 👍🏾

**Pull request merged**